### PR TITLE
545 module loader

### PR DESCRIPTION
**Pull request by NickBurrell**
Implement first pass of module loader. Add support to load module from zip. Import assemblies and textual assets.

### CODE REVIEW

File: engine/CoreEngine/Assets/Scripts/Engine/Objects/Objects/Field.cs [hunk 1]
```diff
@@ -0,0 +1,3 @@
+namespace Synthesis.Core.Object {
+  
+}
```
Comment (matthew17754) [inline engine/CoreEngine/Assets/Scripts/Engine/Objects/Objects/Field.cs:3]: ? 🤨

File: engine/CoreEngine/ProjectSettings/ProjectVersion.txt [hunk 2]
```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2019.4.0f1
-m_EditorVersionWithRevision: 2019.4.0f1 (0af376155913)
+m_EditorVersion: 2019.4.1f1
+m_EditorVersionWithRevision: 2019.4.1f1 (e6c045e14e4e)
```
Comment (matthew17754) [inline engine/CoreEngine/ProjectSettings/ProjectVersion.txt:2]: THIS

**Comment by copernicium** [comment]
These CI builds are failing because of a missing reference to the API dll during the CI build

**Comment by matthew17754** [comment]
Screw it. Add the `.dll`

**Pull request merged**